### PR TITLE
Add get_base_location function and composer test to travis

### PR DESCRIPTION
**Pull request by sebastijansakac**

### CODE REVIEW

File: wirecard-woocommerce-extension/Test/stubs/functions.php [hunk 1]
```diff
@@ -41,6 +41,10 @@ function __( $text, $domain = 'default' ) {
 	return $text;
 }
 
+function wc_get_base_location() {
+	return array("country" => "Austria");
```
Comment (tomazpu) [inline wirecard-woocommerce-extension/Test/stubs/functions.php]: Hi, in the repository we usually declare arrays with a single comma ('). If you can please update it.

Comment (sebastijansakac) [reply wirecard-woocommerce-extension/Test/stubs/functions.php]: Hi, It is updated.

File: wirecard-woocommerce-extension/Test/stubs/functions.php [hunk 2]
```diff
@@ -41,6 +41,19 @@ function __( $text, $domain = 'default' ) {
...
+	return array('country' => 'Austria');
+}
+
+function load_plugin_textdomain() {
+	return true;
+}
+
+function register_activation_hook( $file, $function ) {
+	$file = plugin_basename( $file );
+	add_action( 'activate_' . $file, $function );
```
Comment (tomazpu) [inline wirecard-woocommerce-extension/Test/stubs/functions.php]: please remove the line $file = plugin_basename( $file ); and just use it in add_action with the plugin_basename() function.

File: wirecard-woocommerce-extension/Test/stubs/wp-functions.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?php
+
```
Comment (tomazpu) [inline wirecard-woocommerce-extension/Test/stubs/wp-functions.php]: License header missing.

File: wirecard-woocommerce-extension/woocommerce-wirecard-definitions.php [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?php
+
```
Comment (tomazpu) [inline wirecard-woocommerce-extension/woocommerce-wirecard-definitions.php]: License header missing

File: wirecard-woocommerce-extension/Test/stubs/wp-functions.php [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+{
+	return __DIR__ . '/../../';
+}
+
+function plugin_dir_url()
+{
+	return __DIR__ . '/../';
+}
+
+function plugin_basename()
```
Comment (tomazpu) [inline wirecard-woocommerce-extension/Test/stubs/wp-functions.php]: in the define the method is called plugin_basename( __FILE__ ), with a parameter the stub has no parameter, and the IDE is saying that the definition is wrong. Please correct it.

Same for plugin_dir_path() and plugin_dir_url()

File: wirecard-woocommerce-extension/Test/stubs/wp-functions.php [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+function plugin_dir_path()
```
Comment (tomazpu) [inline wirecard-woocommerce-extension/Test/stubs/wp-functions.php]: Why are those methods not in the file functons.php ? is there a reason?

Comment (sebastijansakac) [reply wirecard-woocommerce-extension/Test/stubs/wp-functions.php]: WP and WC functions are now separated into different files

**Pull request merged**